CryptoTechReminderSystem.AcceptanceTest/CryptoTechReminderSystemTests.cs

```diff
@@ -17,11 +18,11 @@ public class CryptoTechReminderSystemTests
         private const string SlackApiUsersPath = "api/users.list";
         private const string SlackApiPostMessagePath = "api/chat.postMessage";
         private const string HarvestApiUsersPath = "/api/v2/users";
-        private FluentSimulator _slackApi;
-        private FluentSimulator _harvestApi;
-        private HarvestGateway _harvestGateway;
-        private SlackGateway _slackGateway;
-        private RemindDeveloper _remindDeveloper;
+        private static FluentSimulator _slackApi;
+        private static FluentSimulator _harvestApi;
+        private static HarvestGateway _harvestGateway;
+        private static SlackGateway _slackGateway;
+        private static RemindDeveloper _remindDeveloper;
```

Comment:
This use case could be renamed, maybe "SendReminder" would be more appropriate

Reply:
static ?

Reply:
this is the same as `@@` in ruby

Reply:
> static ?

We used them for a one time setup within a text fixture class that is nested within this class - it was an easier way to avoid repetition with the tests

Reply:
does it have to be static?

Reply:
just make everything static

Reply:
Yes, as we wanted to make the variables available in the test fixture.